core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -9,3 +9,4 @@ registry.svc.ci.openshift.org/openshift-kni/4.5:sctptester quay.io/openshift-kni
 registry.svc.ci.openshift.org/openshift-kni/4.5:cnftest-utils quay.io/openshift-kni/cnftest-utils:4.5 quay.io/openshift-kni/cnftest-utils:4.5.0 quay.io/openshift-kni/cnftest-utils:latest
```

Comment:
Can we also bump this one too?

Reply:
For test images I'd wait until there's real need to use a different version in 4.6